src/auto_archiver/storages/gd.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, config: dict) -> None:
             else:
                 logger.debug('GD OAuth Token valid')
         else:
-            gd_service_account = config.service_account
+            gd_service_account = config['gdrive_storage']['service_account']
```

Comment:
```suggestion
            gd_service_account = self.service_account
```
This is actually the standard way of accessing the `configs` via `self`.